Changes

```diff
@@ -9,6 +9,14 @@ Next version (4.05.0):
 
 ### Runtime system:
 
+* GPR#70:
+  The runtime can now be shut down gracefully by means of the new
+  caml_shutdown and caml_startup_pooled functions. The new 'c' flag in
+  OCAMLRUNPARAM enables shutting the runtime properly on process exit.
+  (Max Mouratov, review and discussion by Damien Doligez, Gabriel Scherer,
```

Comment:
Yes, absolutely. But since I'm such a humble and classy person, I prefer to reduce the ungodly power of my true name while venturing into the latin-centric world, in order to... um... blend seamlessly with the common folk I guess :-P

As for you, Mr. Bünzli, please bear your umlaut proudly, as it's not only adorable, but also the best ad for your excellent Unicode libraries :-)